apps/src/code-studio/components/progress/TeacherUnitOverview.tsx

```diff
@@ -233,11 +235,16 @@ const initializeRedux = (
     unitData.announcements.forEach(announcement =>
       dispatch(addAnnouncement(announcement))
     );
+  } else {
+    dispatch(clearAnnouncements());
   }
 
-  if (unitData.student_detail_progress_view) {
```

Comment:
It seems like this is no longer relevant on this page (since we removed the student selector on the page) so I removed it.  Let me know if that tracks with your thinking too.

Reply:
@lfryemason just wanted to draw your attention to this removal.   In a different [comment you said we may need to keep it](https://github.com/code-dot-org/code-dot-org/pull/61794#discussion_r1803648074).  I tested this manually with the experiment and without and didn't see any issues.  I also actually don't see this used in the `UnitOverview` either... 

I just wanted to see if you had a specific spot where you thought this info was needed that perhaps I was missing to make it compatible with the original.

Reply:
We are going to be re-adding a student selector to replace removing the teacher panel and I may have thought this was relevant due to it being named after students. However 1) it's only used for a student looking at the page, which the TeacherUnitOverview will never have and 2) you are right, the value never seems to be read

Thanks for double checking though!